src/types/utils/index.d.ts

```diff
@@ -0,0 +1,122 @@
+declare type ScreenSize = 'full' | 'lg' | 'md' | 'sm'
+
+declare type StatusKind = 'info' | 'error' | 'warning' | 'success'
+
+declare type ButtonKind = 'primary'
+  | 'secondary'
+  | 'accent'
+  | 'danger'
+  | 'ghost'
+  | 'inverse--primary'
+  | 'inverse'
+  | 'danger--primary'
+  | 'tertiary'
+  | 'navigation'
+
+declare type InputModes = 'none'
+ | 'text'
+ | 'tel'
+ | 'url'
+ | 'email'
+ | 'numeric'
+ | 'decimal'
+ | 'search'
+ | undefined
+
+declare interface IIcon  {
```

Comment:
Hi @4m-world, Thank you for your contribution once again. 
Concerning this interface IIcon. Do you mean Icon? If not can you please point me to which type/component will reference the IIcon interface.